### PR TITLE
CloudWatchLogs Scolling and loading improvments

### PR DESCRIPTION
**Pull request by hunterwerlla**
##  Description
- When loading logs around, it automatically scrolls to and selects the entry after it loads
- Fix not loading all the entries when loading logs around by using the paginator
- Refresh now properly sets the table as busy and scrolls to the top after it finishes
- Show seconds and ms in log stream view
- Make table loading faster by not setting items in EDT and instead firing the property changed listener after the items are set
- Fix the empty Log Streams table message to state what the table actually contains
## Screenshots
Kapture 2020-04-02 at 10 24 06

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogStreamActor.kt [hunk 1]
```diff
@@ -70,21 +70,35 @@ sealed class LogStreamActor(
...
+                    // make sure the scroll pane is at the top after loading. Needed for Refresh!
+                    val rect = table.getCellRect(0, 0, true)
+                    table.scrollRectToVisible(rect)
+                }
+                is Message.LOAD_INITIAL_RANGE -> {
+                    loadInitialRange(message.previousEvent.timestamp, message.duration)
+                    val item = table.listTableModel.items.firstOrNull { it == message.previousEvent }
+                    val index = table.listTableModel.indexOf(item).takeIf { it > 0 } ?: return
+                    table.setRowSelectionInterval(index, index)
+                    val rect = table.getCellRect(index, 0, true)
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogStreamActor.kt:84]: com.intellij.ui.TableUtil#scrollSelectionToVisible will scroll to visible spot

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogStreamActor.kt [hunk 2]
```diff
@@ -70,21 +70,35 @@ sealed class LogStreamActor(
...
         try {
+            tableLoading()
             val items = loadBlock()
-            withContext(edtContext) {
-                table.listTableModel.items = items
-            }
+            table.listTableModel.items = items
             table.emptyText.text = emptyText
         } catch (e: ResourceNotFoundException) {
+            table.isVisible = false
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogStreamActor.kt:101]: Do we need to make visible at like 96? Not 100% sure how this interacts with the refresh action

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogStreamActor.kt [hunk 3]
```diff
@@ -186,16 +208,23 @@ class LogStreamListActor(
...
-            .startFromHead(true)
-            .startTime(startTime - duration.toMillis())
-            .endTime(startTime + duration.toMillis())
-            .build()
-        loadAndPopulate { getLogEvents(request, saveForwardToken = true, saveBackwardToken = true) }
+    override suspend fun loadInitialRange(startTime: Long, duration: Duration) = loadAndPopulate {
+        val events = mutableListOf<LogStreamEntry>()
+        client.getLogEventsPaginator {
+            it
+                .logGroupName(logGroup)
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogStreamActor.kt:215]: not: combine with previous line

File: resources/resources/software/aws/toolkits/resources/localized_messages.properties [hunk 4]
```diff
@@ -569,10 +569,10 @@ cloudwatch.logs.open=View Log Streams
 cloudwatch.logs.no_events=No Events in Log Stream
 cloudwatch.logs.no_events_query=No Events matching the query found in Log Stream {0}
 cloudwatch.logs.log_stream_title=Stream: {0}
-cloudwatch.logs.filtered_log_stream_title=Filtered Stream: {0} from {1} to {2}
+cloudwatch.logs.filtered_log_stream_title=Filtered Stream: {0} from {1} to {2} around message "{3}"
```
Comment (abrooksv) [inline resources/resources/software/aws/toolkits/resources/localized_messages.properties:572]: Wonder if message is going to be too long here... Whats it look like?

Comment (hunterwerlla) [reply resources/resources/software/aws/toolkits/resources/localized_messages.properties]: Oh yeah this message is way too long, oh well at least we can have the time there

**Pull request merged**